.github/workflows/analysis_workflow.yml

```diff
@@ -63,7 +67,6 @@ jobs:
           configurePreset: ${{env.ARCTIC_CMAKE_PRESET}}
```

Comment:
Since you're doing a debug build, please _do not_ use `sccache`. Otherwise, this build will end up wiping out the tiny cache space we have.